src/language-js/parse/postprocess/index.js

```diff
@@ -6,6 +6,22 @@ const { isTypeCastComment } = require("../../comments.js");
 const visitNode = require("./visitNode.js");
 const { throwErrorForInvalidNodes } = require("./typescript.js");
 
+const tsKeywordTypes = new Set([
```

Comment:
Use a set is good, but I changed print logic to handle future types in printer before, maybe we should reuse the same logic? https://github.com/prettier/prettier/blob/2213e1719691b3f2da9dc7f00f1c9f0f8d0ae54a/src/language-js/print/typescript.js#L61 

I'm not on my laptop, I didn't checked, but I remember we have a function `isSimpleType` or something, maybe should do the same too.

Reply:
Yes we have `isSimpleType` function. I know `isSimpleType` but I didn't know about `if(node.type.endsWith("Keyword")) {...}`...

Reply:
Here is the context https://github.com/prettier/prettier/pull/10869#issuecomment-840688577